src/utils/presets/utils.js

```diff
@@ -17,17 +17,25 @@ export const dandelionVotingContract = new ethers.utils.Interface(DandelionVotin
 
 const ONE_DAY = 60 * 60 * 24
 
-export const prepareInflationData = (amount) => {
+export const getEthPNTAsset = () => {
   const ethPNTAsset = settings.assets.find((asset) => asset.symbol == 'ethPNT')
   if (!ethPNTAsset) throw new Error('ethPNT asset config not found!')
+  return ethPNTAsset
+}
+
+export const getRawAmount = (amount, decimals) =>
+  BigNumber(amount)
```

Comment:
Why don't we try dropping BigNumber in favour of BigInt?

Reply:
Still using ethers in dao v1.